recipes/pytest-flake8/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pytest-flake8" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aa10a6db147485d71dad391d4149388904c3072194d51755f64784ff128845fd
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - configparser  # [py27]
+    - enum34  # [py27]
+    - flake8 >=3.0
+    - pytest >=2.8
+
+test:
```

Comment:
I've added `pytest --flake8 --help`. Is there a better way to test the entrypoint?